store/postgres/src/store.rs

```diff
@@ -233,9 +233,10 @@ impl BasicStore for Store {
                 .offset(range.skip as i64);
         }
 
-        {
-            debug!(self.logger, "find"; "Postgres query" => format!("{:?}", debug_query::<Pg, _>(&diesel_query)));
-        }
+
+        debug!(self.logger, "find";
+            "sql" => format!("{:?}", debug_query::<Pg, _>(&diesel_query)));
```

Comment:
I'm not clear on how the alignment is off here?  I'm using a single indent on the second line.

Reply:
```
        debug!(self.logger, "find";
            "sql" => format!("{:?}", debug_query::<Pg, _>(&diesel_query)));
```
vs.
```
        debug!(self.logger, "find";
               "sql" => format!("{:?}", debug_query::<Pg, _>(&diesel_query)));
```
which one is aligned correctly?

Reply:
nm. I see how it needs to be formatted.